src/middlewares/multipartFormDataParser.js

```diff
@@ -0,0 +1,49 @@
+const BusBoy = require('busboy')
+const contentTypeLib = require('content-type')
+const createError = require('http-errors')
+
+module.exports = () => ({
+  before: async (handler, next) => {
+    const { headers } = handler.event
+    if (!headers) {
+      return next()
+    }
+
+    const contentType = headers['content-type']
+    if (contentType) {
+      const { type } = contentTypeLib.parse(contentType)
+      if (type === 'multipart/form-data') {
```

Comment:
We should handle the case when the type is not what we expect and just call `next()`

I would prefer a reverse logic and do an early return as you already did in lines `8-10`. Something like:

```javascript
if (type !== 'multipart/form-data') {
  return next()
}
```

Reply:
Sounds good. Implemented.

Reply:
Tnx